### PR TITLE
3529: Port privacy notice page

### PR DESCRIPTION
**Pull request by oswaldquek**
English and Welsh have static pages

solo: @oswaldquek

### CODE REVIEW

**Comment by oswaldquek** [comment]
DO NOT MERGE YET

File: spec/features/user_visit_privacy_notice_page_spec.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  it 'displays the page in English' do
+    visit '/privacy-notice'
+    expect(page).to have_title 'Privacy notice - GOV.UK Verify - GOV.UK'
+    expect(page).to have_link 'Summary', href: '#summary'
+    expect(page).to have_content 'GOV.UK Verify is provided by the Government Digital Service (GDS). We use your personal data to help you to transact securely with government services.'
+    expect(page).to have_link 'Privacy policies of certified companies and government departments', href: '#privacy-policies-of-certified-companies-and-government-departments'
+    expect(page).to have_content 'The certified companies have their own privacy policies that apply to their handling of your personal data as part of the identity verification process. Likewise, the government services have their own privacy policies that apply to their handling of your personal data when you use their services.'
+  end
+
+  it 'displays the page in Welsh' do
```
Comment (willp-bl) [inline spec/features/user_visit_privacy_notice_page_spec.rb]: could mark this as pending, or check the page's locale

**Comment by willp-bl** [comment]
This needs to be rebased against master

File: spec/features/user_visit_privacy_notice_page_spec.rb (hunk 1 above)
Comment (oswaldquek) [inline spec/features/user_visit_privacy_notice_page_spec.rb]: we're already visiting /privacy-notice-cy in the test and testing for the Welsh banner appearing so i think this is ok

File: app/assets/stylesheets/elements/_elements-typography.scss [hunk 2]
```diff
@@ -286,3 +286,9 @@ hr {
     margin-left: -$gutter-half;
   }
 }
+
```
Comment (robinwhittleton) [inline app/assets/stylesheets/elements/_elements-typography.scss]: We don’t want to be editing/amending anything in this folder: it’ll be overridden the next time govuk_elements is updated. Can this be moved to the overrides file?

File: app/views/privacy_notice/index.cy.html.erb [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+      </li>
+      <li id="#certified-company"><em>Certified company:</em> When you use GOV.UK Verify to access a government service,
+        you choose from a list of companies certified to verify your identity before confirming this to the government
+        service you&rsquo;re trying to use. Certified companies are companies that have been certified as meeting
+        government and industry-agreed protocols and standards for identity assurance and information security and have
+        completed a
+        <a href="https://identityassurance.blog.gov.uk/2015/07/17/gov-uk-verify-making-sure-certified-companies-are-ready-to-connect/">rigorous
+          onboarding process</a>.
+        <a href="https://www.gov.uk/government/publications/introducing-govuk-verify/introducing-govuk-verify#certified-companies">We
+          publish a full list of certified companies</a>.
```
Comment (robinwhittleton) [inline app/views/privacy_notice/index.cy.html.erb]: Missing an `</li>` here.

File: app/views/privacy_notice/index.cy.html.erb [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+      transact securely with government services.</strong></p>
+    <p>GOV.UK Verify allows you to access <a href="#government-services">government services</a> securely. You use
+      GOV.UK
+      Verify to pick a <a href="#certified-company">certified company</a> which verifies your identity. Once your
+      identity
+      has been verified, the certified company passes the verified information back to GOV.UK Verify, which sends this
+      information to the government service you are trying to access.</p>
+    <p>This privacy notice explains what information we might collect, how it’s used, how it’s protected and how you can
+      find out about it.</p>
+    <h2 class="heading-medium" id="#privacy-policies-of-certified-companies-and-government-departments">Privacy policies
```
Comment (robinwhittleton) [inline app/views/privacy_notice/index.cy.html.erb]: `id` attributes shouldn’t start with `#` - can this be removed here and on the other `h2`s with the same problem?

File: app/views/privacy_notice/index.cy.html.erb [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+      <li>At your request, your certified company will send the verified data about you (your name, address and date of
+        birth along with your gender if you choose to provide it) to the GOV.UK Verify Hub, and the Hub will send it to
+        the government service that you want to access. The certified company you use won’t know which government
+        service
+        has requested this information. Likewise, the government service will not know which certified company you have
+        chosen.
+      </li>
+      <li>Providing a <a href="#document-checking-service">Document Checking Service (DCS)</a> to certified companies,
+        that allows them to check the validity of your UK passport or driving licence with HM Passport Office (HMPO) and
+        the Driver & Vehicle Licensing Agency (DVLA). The data that HMPO and DVLA hold about you is not shared with the
```
Comment (robinwhittleton) [inline app/views/privacy_notice/index.cy.html.erb]: `&` should be `&amp;`.

File: app/views/privacy_notice/index.cy.html.erb [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+      <a href="https://identityassurance.blog.gov.uk/">GOV.UK Verify Blog</a>.
+    <h2 class="heading-medium" id="#asking-to-see-your-personal-data">Asking to see your personal data</h2>
+    <p>If you wish to see personal data that’s collected by a certified company or by a government service, you need to
+      contact them directly.</p>
+    <p>You can also ask to see the personal information that we hold about you. To do so, please email or write to us.
+      There’s no charge for information provided. In very limited circumstances, we may withhold some information in
+      accordance with Part II of the Data Protection Act 1998.</p>
+    <h2 class="heading-medium" id="#how-to-make-a-complaint">How to make a complaint</h2>
+    <p>If you’re unhappy with the way GOV.UK Verify handles your personal data, you can email or write to us. You’ll get
+      a
```
Comment (robinwhittleton) [inline app/views/privacy_notice/index.cy.html.erb]: Line breaks here are a little awkward. Can the text go onto a single line?

**Comment by robinwhittleton** [comment]
Can you run the finished page through http://validator.w3.org/ when it’s ready to double-check that the markup errors have been caught? Thanks.

File: app/views/privacy_notice/index.cy.html.erb (hunk 6 above)
Comment (oswaldquek) [inline app/views/privacy_notice/index.cy.html.erb]: It was auto formatted by RubyMine when I alt+cmd+l'ed!

**Pull request closed without merging**